Kenny/docs rewrite (#44)
latchbio/latch: docs/source/basics/writing_dockerfiles.md

Writing Dockerfiles
===

> Dockerfiles allow you to define computing environment that your task will

Reviewer: ```suggestion
Dockerfiles allow you to define the computing environment that your task will
```

execute in.

> To write a Dockerfile, you simply write commands that you want executed

Reviewer: ```suggestion
To write a Dockerfile, you simply write the commands that you want executed
```

> or specify files that you want available before your task is run. A Dockerfile

Reviewer: ```suggestion
and specify the files that you want available before your task is run. A Dockerfile
```

> defines your task's "image" (the recipe for its virutal computing environment)

Reviewer: ```suggestion
defines your task's "image" (the recipe for its virtual computing environment)
```
